### PR TITLE
Observability + metrics on Dedicated

### PR DESCRIPTION
**Pull request by chadwcarlson**
Adds observability/metrics page for Dedicated.

### CODE REVIEW

**Comment by chadwcarlson** [comment]
Screen Shot 2021-03-11 at 3 14 21 PM

Before I delete the test shortcode, here's a screenshot verifying that the check is working fine on the environment. It's difficult to debug otherwise, since GTAM will not appear on dev environments normally.

File: docs/src/dedicated/architecture/metrics.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+| 5 - 24 hours                 | 20 minutes                        |
+
+From the taskbar you can also filter out individual hosts. 
+
+![Metrics - sampling](/images/management-console/metrics/sampling.png "0.4")
+
+Standard Dedicated environments will have a single cluster of three hosts, with each additional cluster adding at least three additional hosts to the project. Dedicated environments with two clusters have a split architecture, and their metrics are displayed in two separate groups: one for the web hosts that handle web requests and one for the service hosts that handle database, cache, and other services. 
+
+![Metrics - sampling](/images/management-console/metrics/split-arch.png "0.75")
+
```
Comment (MariadeAnton) [inline docs/src/dedicated/architecture/metrics.md:45]: @chadwcarlson do we need a main header here that groups what comes after this point? 

Feels like a great overview on the data we display along with an example for CPU, RAM and disk.

File: docs/src/other/changelog.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ description: |
 
 ## 2021
 
+* **March 2021**
```
Comment (MariadeAnton) [inline docs/src/other/changelog.md:12]: The changelog is great, thank you for this @chadwcarlson !

File: docs/layouts/partials/scripts/user-customizations.html [hunk 3]
```diff
@@ -3,9 +3,11 @@
 <script src="{{"scripts/user-widget.js" | relURL}}"></script>
 
 <!-- Google Tag Manager -->
+{{- if gt ( len ( getenv "PLATFORM_PROJECT" ) ) 0 -}}
```
Comment (chadwcarlson) [inline docs/layouts/partials/scripts/user-customizations.html:6]:
```suggestion
{{- if and ( gt ( len ( getenv "PLATFORM_PROJECT" ) ) 0 ) ( ne "master"  ( getenv "PLATFORM_BRANCH" )) -}}
```

Comment (chadwcarlson) [reply docs/layouts/partials/scripts/user-customizations.html:6]: This ain't gonna work, since `PLATFORM_BRANCH` isn't available during builds....

**Comment by chadwcarlson** [comment]
Testing integration, will re-open shortly.

**Review by MariadeAnton — APPROVED** [review]

**Pull request merged**